README.md

```diff
@@ -17,7 +36,8 @@ git push -u origin main
 ```
```

Comment:
Could we add instructions for git submodule init/update here too?

Reply:
Also, personally I needed to make a docker-compose override to get this project spun up. Did you? Should that be required/mentioned in the readme?